### PR TITLE
Anders/ansible automation

### PR DESCRIPTION
**Pull request by andersla**
@mcapuccini Here you go

### CODE REVIEW

**Comment by mcapuccini** [comment]
[One click deployment of infrastructure](https://trello.com/c/mfkjMZVv/5-one-click-deployment-of-infrastructure)

File: ansible.cfg [hunk 1]
```diff
@@ -1,6 +1,8 @@
 [defaults]
 host_key_checking=False
 hostfile=KubeNow/inventory
+# Note: You have to specify full path to the key file, otherwise silently ignored
+private_key_file=vre.key
```
Comment (mcapuccini) [inline ansible.cfg:4]: this gets generated automatically, right?

Comment (mcapuccini) [reply ansible.cfg:4]: yep... just saw presetup!

**Comment by andersla** [comment]
yep in presetup.sh
      
      On 2016-12-06 10:56, Marco Capuccini wrote:
    
    
      @mcapuccini commented on this pull request.
      
      In ansible.cfg:
      > @@ -1,6 +1,8 @@
 [defaults]
 host_key_checking=False
 hostfile=KubeNow/inventory
+# Note: You have to specify full path to the key file, otherwise silently ignored
+private_key_file=vre.key

      this gets generated automatically, right?
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/phnmnl/MTBLS233-POP","title":"phnmnl/MTBLS233-POP","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/phnmnl/MTBLS233-POP"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mcapuccini commented on #9"}],"action":{"name":"View Pull Request","url":"https://github.com/phnmnl/MTBLS233-POP/pull/9#pullrequestreview-11553787"}}}

**Comment by mcapuccini** [comment]
[Use Luigi/Swift instead of Spark/HDFS](https://trello.com/c/5GgumuWL/4-use-luigi-swift-instead-of-spark-hdfs)

**Comment by mcapuccini** [comment]
@andersla awesome! I made some mods, give it a look please. I am merging now.

**Pull request merged**